src/components/Footer/style.ts

```diff
@@ -17,7 +17,9 @@ export const FooterContainer = styled.footer`
 
   @media (min-height: 1000px) {
     position: absolute;
-    top: calc(100vh - 5rem);
+    top: calc(
+      100vh - 5rem
+    ); // 화면 전체에서 맨 아래에 footer가 위치되도록 -> 화면 높이 - footer 높이
```

Comment:
```suggestion
    bottom: 0;
```

이렇게 대체 가능해보입니다
제가 간과한 부분이 있는지 확인 한번 부탁드립니다

Reply:
괜찮은것 같습니다!
잡아주셔서 감사드립니다